frontend/public/components/resource-list.tsx

```diff
@@ -8,12 +9,23 @@ import { connectToPlural } from '../kinds';
 import { LoadingBox } from './utils';
 import { K8sResourceKindReference } from '../module/k8s';
 import { ErrorPage404 } from './error';
+import { FLAGS, connectToFlags } from '../features';
 
 // Parameters can be in pros.params (in URL) or in props.route (attribute of Route tag)
 const allParams = props => Object.assign({}, _.get(props, 'match.params'), props);
 
-export const ResourceListPage = connectToPlural((props: ResourceListPageProps) => {
-  const { ns, kindObj, kindsInFlight } = allParams(props);
+export const OpenShiftGettingStarted = () => <div className="co-well">
```

Comment:
Should this be imported from an `openshift-x.tsx`or `getting-started.tsx` kind of file?  Since this file is `resource-list.tsx`, seems like we should keep it clean & avoid cluttering resource-list.

Reply:
Yeah, I don't mind moving this into it's own file. It could be used in other places, so it might make sense to do that.